### PR TITLE
Do not block when managed files should be moved

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def step5(self, step):
             if action == 'remove':
                 self.files_to_remove(upgrade_file)
             elif action == 'move':
-                task_to_do |= self.files_to_move(upgrade_file)
+                task_to_do |= self.files_to_move(upgrade_file, force=True)
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: This will hide the tasks to do but it shouldn't...

**Comment by sbrunner** [comment]
@arnaud-morvan are you working on this pull request? Can I purpose an alternative?

**Comment by arnaud-morvan** [comment]
I'm working on WMS Browser search, and I need to follow GetItFixed, so I'm pretty sure I won't work on c2cupgrade this week, feel free to assign it to yourself.

**Comment by arnaud-morvan** [comment]
Done in #5329, closing

**Pull request closed without merging**